### ISSUE TITLE
Move Form\Validator class to Validator\

### ISSUE THREAD

**Issue by eddiejaoude** [TransformCore/CSR-Fast-Stream-Domain-Model#34]
Move
`Bundle\Form\Validator\Applicants`

To
`Bundle\Validator\...`

**Comment by eddiejaoude** [comment]
And move from `static`

**Comment by knasher** [comment]
The Applicants validator is no longer needed since change password has been moved to a separate page, you will see that this has therefore been removed in CSR-344. Once this is merged in you can close this issue.

**Comment by eddiejaoude** [comment]
ok thanks!
